doc/source/contributing_docs.rst

```diff
@@ -169,52 +284,47 @@ environmental variable like this::
 Adding changes to the UserGuide
 ===============================
 
-As with the code, :ref:`commit and push <adding-code-to-mda>` your code to GitHub. Then :ref:`create a pull request <create-a-pull-request>`. The only test run for the User Guide is: that your file compile into HTML documents without errors. As usual, a developer will review your PR and merge the code into the User Guide when it looks good.
+As with the code, :ref:`commit and push <adding-code-to-mda>` your code to GitHub.
+Then :ref:`create a pull request <create-a-pull-request>`.
+The only test run for the User Guide is: that your file compile into HTML documents without errors.
+As usual, a developer will review your PR and merge the code into the User Guide when it looks good.
```

Comment:
```suggestion
As usual, your PR will be reviewed and merged into the User Guide when it looks good.
```
Just to be a bit more open about who can/will do this? I.e. a non-developer but technical writer, etc.